lib/runners/harness.rb

```diff
@@ -91,5 +92,15 @@ def handle_error(exn)
         trace_writer.error "#{exn.message} (#{exn.class})"
       end
     end
+
+    def remove_ignored_files(config)
+      globs = Array(config.content[:ignore])
+      return if globs.empty?
+
+      trace_writer.header("Deleting specified files via the `ignore` option: #{globs.join(', ')}")
+      list = globs.flat_map { |g| Dir[working_dir / g] }
+      message = list.map { |l| Pathname(l).relative_path_from(working_dir) }.join("\n")
+      trace_writer.message(message) { FileUtils.rm_rf(list) }
```

Comment:
What about the following simple solution?

```suggestion
      trace_writer.message "Deleting files via the `ignore` option..." do
        FileUtils.rm_rf globs
        trace_writer.message globs.join("\n")
      end
```

- `trace_writer.header` looks better to me if under "Setting up source code".

<img width="649" alt="image" src="https://user-images.githubusercontent.com/473530/75643439-68160980-5c82-11ea-8768-f5d47b8c8d18.png">

Reply:
Sorry, the following is correct 🙏 

```suggestion
      trace_writer.message "Deleting files via the `ignore` option..." do
        files = Pathname(working_dir).glob(globs)
        FileUtils.rm_rf files
        trace_writer.message files.map { |f| f.relative_path_from(working_dir) }.join("\n")
      end
```

Reply:
Fixed with 944b7df